src/personas/maid_manager/mod.rs

```diff
@@ -703,23 +858,69 @@ impl MaidManager {
     pub fn get_mutation_count(&self, client_name: &XorName) -> Option<u64> {
         self.accounts
             .get(client_name)
-            .map(|account| account.info.mutations_done)
+            .map(|account| account.data_ops_msg_ids.len() as u64)
     }
 }
 
 #[derive(Deserialize, Serialize, PartialEq, Eq, Debug, Clone)]
-enum Refresh {
-    Update(XorName, Account),
+pub enum Refresh {
```

Comment:
The names of the variants are now confusing and do not reflect well their purpose. My suggestion:
- rename `Update` to `UpdateDataOps`
- rename `UpdateData` to `InsertDataOp`

Also for consistency, rename the methods:

- `handle_refresh_update` to `handle_refresh_update_data_ops`
- `handle_refresh_update_data` to `handle_refresh_insert_data_op`

Feel free to suggest different names, but the important thing is to make them descriptive and consistent.